paper-menu-button.html

```diff
@@ -366,9 +371,7 @@
       }
     });
 
-    PaperMenuButton.ANIMATION_CUBIC_BEZIER = 'cubic-bezier(.3,.95,.5,1)';
-    PaperMenuButton.MAX_ANIMATION_TIME_MS = 400;
-
+    PaperMenuButton.CONFIG = config;
```

Comment:
Instead of creating the new `CONFIG` namespace, please update this so that the same constants will be created where they used to exist. e.g., `PaperMenuButton.ANIMATION_CUBIC_BEZIER = config.ANIMATION_CUBIC_BEZIER`.